.github/workflows/cd-dashboard.yml

```diff
@@ -57,6 +78,6 @@ jobs:
             --region ${{ env.REGION }} \
             --image gcr.io/${{ env.PROJECT_ID }}/${{ env.SERVICE }}:${{  github.sha }} \
             --platform "managed" \
-            --set-env-vars NEXT_PUBLIC_SIDETREE_METHOD="${{ env.NEXT_PUBLIC_SIDETREE_METHOD }}",NEXT_PUBLIC_OPERATOR="${{ env.NEXT_PUBLIC_OPERATOR }}",NEXT_PUBLIC_METHOD="${{ env.NEXT_PUBLIC_METHOD }}",NEXT_PUBLIC_DESCRIPTION="${{ env.NEXT_PUBLIC_DESCRIPTION }}",MONGO_DB_CONNECTION_STRING="${{ env.MONGO_DB_CONNECTION_STRING }}",DATABASE_NAME="${{ env.DATABASE_NAME }}",MAX_CONCURRENT_DOWNLOADS="${{ env.MAX_CONCURRENT_DOWNLOADS }}",BATCH_INTERVAL_IN_SECONDS="${{ env.BATCH_INTERVAL_IN_SECONDS }}",OBSERVING_INTERVAL_IN_SECONDS="${{ env.OBSERVING_INTERVAL_IN_SECONDS }}",ELEMENT_CONTENT_ADDRESSABLE_STORE_SERVICE_URI="${{ env.ELEMENT_CONTENT_ADDRESSABLE_STORE_SERVICE_URI }}",ELEMENT_ANCHOR_CONTRACT="${{ env.ELEMENT_ANCHOR_CONTRACT }}",ETHEREUM_RPC_URL="${{ env.ETHEREUM_RPC_URL }}",ETHEREUM_PROVIDER="${{ env.ETHEREUM_PROVIDER }}",ETHEREUM_MNEMONIC="${{ env.ETHEREUM_MNEMONIC }}" \
+            --set-env-vars NEXT_PUBLIC_SIDETREE_METHOD="${{ env.NEXT_PUBLIC_SIDETREE_METHOD }}",NEXT_PUBLIC_OPERATOR="${{ env.NEXT_PUBLIC_OPERATOR }}",NEXT_PUBLIC_METHOD="${{ env.NEXT_PUBLIC_METHOD }}",NEXT_PUBLIC_DESCRIPTION="${{ env.NEXT_PUBLIC_DESCRIPTION }}",NEXT_PUBLIC_LOGO_TEXT_DARK="${{ env.NEXT_PUBLIC_LOGO_TEXT_DARK }}",NEXT_PUBLIC_LOGO_TEXT_LIGHT="${{ env.NEXT_PUBLIC_LOGO_TEXT_LIGHT }}",MONGO_DB_CONNECTION_STRING="${{ env.MONGO_DB_CONNECTION_STRING }}",DATABASE_NAME="${{ env.DATABASE_NAME }}",MAX_CONCURRENT_DOWNLOADS="${{ env.MAX_CONCURRENT_DOWNLOADS }}",BATCH_INTERVAL_IN_SECONDS="${{ env.BATCH_INTERVAL_IN_SECONDS }}",OBSERVING_INTERVAL_IN_SECONDS="${{ env.OBSERVING_INTERVAL_IN_SECONDS }}",ELEMENT_CONTENT_ADDRESSABLE_STORE_SERVICE_URI="${{ env.ELEMENT_CONTENT_ADDRESSABLE_STORE_SERVICE_URI }}",ELEMENT_ANCHOR_CONTRACT="${{ env.ELEMENT_ANCHOR_CONTRACT }}",ETHEREUM_RPC_URL="${{ env.ETHEREUM_RPC_URL }}",ETHEREUM_PROVIDER="${{ env.ETHEREUM_PROVIDER }}",ETHEREUM_MNEMONIC="${{ env.ETHEREUM_MNEMONIC }}" \
```

Comment:
These changes do not use the file you are creating.

Reply:
it's used for node server environment, the file is used for client when next js builds

Reply:
but doesn't that mean you are bundling the secrets into the container?